introduction.md

```diff
@@ -1 +1,4 @@
-Javascript is my favorite codeschool path.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?